### PR TITLE
Update logs.md & cluster-resources.md

### PR DESCRIPTION
**Pull request by edgarlanting**
In combination with the PR that fixes Troubleshoot issue 878 (https://github.com/replicatedhq/troubleshoot/issues/878) this PR handles the additional mention of the maximum logfile size.

### CODE REVIEW

**Review by xavpaice — CHANGES_REQUESTED** [review]
Please ensure that the way to change this limit is documented.

Also, please ensure that there's a note in the clusterResources collector, since that defaults to 5MB limit too.

**Comment by xavpaice** [comment]
Thanks for updating the log collector doc.  We still need a note in the clusterResources collector doc to highlight the 5MB limit.

File: docs/source/collect/logs.md [hunk 1]
```diff
@@ -62,6 +62,10 @@ For duration string format see [time.ParseDuration](https://pkg.go.dev/time#Pars
 ##### `limits.maxLines`
 The number of lines to include, starting from the newest.
 
+##### `Limits.maxBytes`
```
Comment (ajp-io) [inline docs/source/collect/logs.md:65]:
```suggestion
##### `limits.maxBytes`
```

File: docs/source/collect/logs.md [hunk 2]
```diff
@@ -62,6 +62,10 @@ For duration string format see [time.ParseDuration](https://pkg.go.dev/time#Pars
...
+The maximum file size of a collected pod log, is currently set to 5MB as this will ensure a support bundle will not grow excessively due
+to large logfiles. Neither will the logs contain information that is too old or no longer relevant.
+
```
Comment (xavpaice) [inline docs/source/collect/logs.md:68]: please include the format for this field - can I specify 10MB or do I need to write that in bytes as an integer?

File: docs/source/collect/logs.md [hunk 3]
```diff
@@ -62,6 +62,11 @@ For duration string format see [time.ParseDuration](https://pkg.go.dev/time#Pars
 ##### `limits.maxLines`
 The number of lines to include, starting from the newest.
 
+##### `limits.maxBytes`
+The maximum file size of a collected pod log, is currently set to 5MB as this will ensure a support bundle will not grow excessively due
+to large logfiles. Neither will the logs contain information that is too old or no longer relevant.
+This value can be set in increments of bytes, e.g. a value of 5MB should be set as `5000000`.
```
Comment (ajp-io) [inline docs/source/collect/logs.md:68]: Just a suggestion, but I don't think all the background about the logs getting too large is needed in the description of what the field does. Seems fairly self-explanatory what you would use it for, and the descriptions of other fields tend to be briefer.
```suggestion
The maximum file size of a collected pod log. Defaults to `5000000`, which is 5MB.
```

Comment (edgarlanting) [reply docs/source/collect/logs.md:68]: Agree, thanks @ajp-io

**Review by ajp-io — DISMISSED** [review]

**Review by xavpaice — CHANGES_REQUESTED** [review]
Two items:

* please add a note to the cluster resources collector docs stating that there is a 5MB limit on pod log size.
* please clarify the format of limits.maxBytes, to ensure that folks know it has to be an integer and they can't use '5MB' or '300KB'.

**Review by xavpaice — APPROVED** [review]

**Comment by xavpaice** [comment]
I've made one minor change, could use a docs review before merging.

**Review by jonquil2002 — APPROVED** [review]
lgtm. Just deleted some extra words ("set to") for minimalism.

**Pull request merged**